<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowData</class>
 <widget class="QWidget" name="ShowData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Home">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>110</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LOCK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ConnectServer">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>110</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_UpdateData">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>110</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上报数据</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>901</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_device1">
    <widget class="QLabel" name="light_intensity_icon_label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/rec/img/light.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Humidity">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="humidity__text_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperture_text_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperture_icon_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/rec/img/temperture.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="light_intensity_text_label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>130</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>光强</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Temperture">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="humidity_icon_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/rec/img/humidity.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Light">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>60</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_device1">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>270</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DEVICE1</string>
     </property>
    </widget>
    <widget class="QLabel" name="device1_light_intensity_text_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>260</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>灯光状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="device1_light_icon_label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>230</y>
       <width>120</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/rec/img/light_off.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_device2">
    <widget class="QLCDNumber" name="lcdNumber_Air">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gas_text_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>气体</string>
     </property>
    </widget>
    <widget class="QDial" name="dial_FanSpeed">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>200</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="gas_icon_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/rec/img/gas.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_FanSpeed">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>220</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_device2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>270</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DEVICE2</string>
     </property>
    </widget>
    <widget class="QLabel" name="device2_light_intensity_text_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>灯光控制</string>
     </property>
    </widget>
    <widget class="QLabel" name="device2_light_icon_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>120</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/rec/img/light_off.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="light_checkBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator:unchecked {
image: url(:/rec/img/close.png);
}
QCheckBox::indicator:checked {
image: url(:/rec/img/open.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fanSpeed_text_label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>320</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>风速</string>
     </property>
    </widget>
    <widget class="QLabel" name="soil_icon_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>90</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/rec/img/soil.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Soil">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>50</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="soil_text_label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>140</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(115, 210, 22);</string>
     </property>
     <property name="text">
      <string>土壤</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_device3">
    <widget class="QLabel" name="label_device3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>270</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>31</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DEVICE3</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_nextPage">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>500</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/img/next_page.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_previousPage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/img/previous_page.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_ConnectServer">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>未连接到服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ReceiveData">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>110</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>接收数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
